python/tvm/relay/backend/graph_runtime_codegen.py

```diff
@@ -85,7 +85,7 @@ def codegen(self, func):
         param_names = self._list_params_name()
         params = {}
         for key in param_names:
-            arr = self._get_param_by_name(key)
+            arr = self._get_param_by_name(str(key))
```

Comment:
This is something that we need to update in GraphRuntimeCodegen instead, to explicit get an String instead of str